internal/ioutil/print.go

```diff
@@ -52,3 +53,30 @@ func PrintCertificateMap(w io.Writer, v []config.CertificateReference) error {
 	}
 	return tw.Flush()
 }
+
+func PrintVerificationResults(w io.Writer, v []*verification.SignatureVerificationOutcome, resultErr error, digest string) error {
+	tw := newTabWriter(w)
+
+	if resultErr == nil {
+		fmt.Fprintf(tw, "%s\n", digest)
+		return nil
+	}
+
+	fmt.Fprintf(tw, "ERROR: %s\n\n", resultErr.Error())
+	printOutcomes(tw, v)
+
+	return tw.Flush()
+}
+
+func printOutcomes(tw *tabwriter.Writer, outcomes []*verification.SignatureVerificationOutcome) {
+	if len(outcomes) == 1 {
+		fmt.Println("1 signature failed verification, error is listed as below:")
+	} else {
+		fmt.Printf("%d signatures failed verification, errors are listed as below:\n", len(outcomes))
+	}
+
+	for _, outcome := range outcomes {
+		// TODO: print out the signature digest once the outcome contains it.
+		fmt.Printf("%s\n\n", outcome.Error.Error())
+	}
```

Comment:
IMO this error message is not really helpful and should only be printed in debug level.

In non-debug mode, we should display some generic message like:
`Signature verification failed for all the xx signatures associated with ${reference}. `

In debug mode
```
Signature verification failed for all the xx signatures associated with ${reference}. ` The failure reasons for each signature are as follows:
1. ${signature-reference}: ${error message}
2.  ${signature-reference}: ${error message}
````

Reply:
@yizha1 Could you help check if you need to update the spec?

Reply:
I can update verify CLI spec for this accordingly

Reply:
Tracking it in https://github.com/notaryproject/notation/issues/304